### PR TITLE
Add "Reload File" feature

### PR DESCRIPTION
**Pull request by jdpurcell**
Implements #587

### CODE REVIEW

File: src/qvimagecore.cpp [hunk 1]
```diff
@@ -68,7 +68,7 @@ QVImageCore::QVImageCore(QObject *parent) : QObject(parent)
     settingsUpdated();
 }
 
-void QVImageCore::loadFile(const QString &fileName)
+void QVImageCore::loadFile(const QString &fileName, bool isReloading)
```
Comment (jdpurcell) [inline src/qvimagecore.cpp:71]: The cache key includes the file size, but this ensures it handles situations in which the size doesn't change (e.g. an uncompressed BMP was re-saved).

File: src/qvimagecore.h [hunk 2]
```diff
@@ -60,7 +60,7 @@ class QVImageCore : public QObject
 
     explicit QVImageCore(QObject *parent = nullptr);
 
-    void loadFile(const QString &fileName);
+    void loadFile(const QString &fileName, bool isReloaing = false);
```
Comment (jurplel) [inline src/qvimagecore.h:63]: Typo here

File: src/shortcutmanager.cpp [hunk 3]
```diff
@@ -44,6 +44,7 @@ void ShortcutManager::initializeShortcutsList()
 {
     shortcutsList.append({tr("Open"), "open", keyBindingsToStringList(QKeySequence::Open), {}});
     shortcutsList.append({tr("Open URL"), "openurl", QStringList(QKeySequence(Qt::CTRL | Qt::SHIFT | Qt::Key_O).toString()), {}});
+    shortcutsList.append({tr("Reload File"), "reloadfile", keyBindingsToStringList(QKeySequence::Refresh), {}});
```
Comment (jurplel) [inline src/shortcutmanager.cpp:47]: indeed this is cmd+r on macOS, but this conflicts with rename. I don't know which should be prioritized--maybe refresh is more important. 

Make sure this doesn't conflict on other platforms too.

**Review by jurplel — CHANGES_REQUESTED** [review]

File: src/qvimagecore.h (hunk 2 above)
Comment (jdpurcell) [reply src/qvimagecore.h]: Fixed!

File: src/shortcutmanager.cpp (hunk 3 above)
Comment (jdpurcell) [reply src/shortcutmanager.cpp:47]: Good catch, you can take a look at my latest commit and see if you like that idea or have a different suggestion perhaps.

**Comment by jurplel** [comment]
Great

**Pull request merged**